modules/assim.sequential/R/Analysis_sda_multiSite.R

```diff
@@ -359,8 +360,14 @@ GEF.MultiSite <- function(settings, Forecast, Observed, H, extraArg,...){
                nitr.GEF = extraArg$nitr.GEF,
                nburnin = extraArg$nburnin,
                nthin = extraArg$nthin,
-               monitors = c("Xall", "qq"))
-  outputs <- furrr::future_map(lapply(rep("data", as.numeric(settings$state.data.assimilation$chains)), get), MCMC_function)
+               monitors = c("Xall", "qq"),
+               t=t)
+  tmp = list()
+  for(i in seq_len(as.numeric(settings$state.data.assimilation$chains))){
+    tmp[[i]] = MCMC_data
+  }
+  MCMC_data = tmp
```

Comment:
Is this equivalent to `MCMC_data = rep(list(MCMC_data), settings$state.data.assimilation$chains)`? That form would make it clearer to me that this is a single list of identical copies of the same list.
I haven't tested, but `rep` is probably more memory efficient too, because it can be allocated once and doesn't need to grow